static/src/vu/variables.scss

```diff
@@ -1,10 +1,13 @@
 // colors

```

Comment:
Can we use the Bootstrap colors in some cases here? For example for `$white` and `$black`?

https://getbootstrap.com/docs/5.3/customize/color/#all-colors